dhall-docs/tasty/data/package/JumpToRecordFieldWhenVarIsOfRecordTypeTransitivity.dhall

```diff
@@ -0,0 +1,17 @@
+{-|
+On this example, `dhall-docs` will detect that the variable `a` is bounded
+to a record literal and it will be aware that it uses dot-syntax.
+
+Transitivity is respected when the variable is assigned using let-bindings.
+For instance, if the user clicks the `y` from the `ax.y.z`, it will jump to
+the definition of `x` on the record literal.
+
+_(the code is the same from <./JumpToRecordFieldWhenVarIsOfRecordTypeWithDotSyntax.dhall>)_
```

Comment:
What is the reason for having duplicate examples?

Reply:
I like to keep one feature or bug-fix per file. Although the file is a duplicate, it is a good example of transitivity.

We can change the sample, if you want to

Reply:
I think at least minimize each example to the smallest code that illustrates what's going on